external/loaders/loaders/batches/_batch.py

```diff
@@ -138,8 +144,9 @@ def batches_from_mapper(
     shuffle_timesteps: bool = True,
     shuffle_samples: bool = False,
     data_transforms: Optional[Sequence[Mapping]] = None,
+    gsrm_name: str = "fv3",
```

Comment:
Same point as above in terms of `res` and `gsrm_name` arguments.

Reply:
Responded above